.tekton/push.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: cachi2-on-push
+  annotations:
+    pipelinesascode.tekton.dev/on-event: "[push]"
+    pipelinesascode.tekton.dev/on-target-branch: "[main]"
+    pipelinesascode.tekton.dev/max-keep-runs: "5"
+spec:
+  params:
+    - name: git-url
+      value: "{{repo_url}}"
+    - name: revision
+      value: "{{revision}}"
+    - name: output-image
+      value: 'quay.io/redhat-appstudio/cachi2:latest'
```

Comment:
Can we keep the image tag for the previous ref somehow? I'm not sure if PR builds (even if we don't clean them up) cover for that. I think the ref changes when you merge the PR

Reply:
Maybe we just don't use the `latest` tag and only tag things with the revision? We'll need to figure out how to tag with semantic versions very soon anyway

Reply:
^ Speaking of golden-container, this is how they tag latest: https://github.com/enterprise-contract/golden-container/blob/main/.tekton/tag-latest.yaml